srcpkgs/distcc/template

```diff
@@ -3,24 +3,24 @@ pkgname=distcc
 version=3.3.3
 revision=6
 build_style=gnu-configure
-configure_args="--disable-Werror --with-gtk"
+configure_args="--disable-Werror"
 conf_files="
 	/etc/distcc/hosts
 	/etc/distcc/clients.allow"
 hostmakedepends="automake libtool pkg-config which"
-makedepends="binutils-devel popt-devel avahi-libs-devel gtk+-devel"
+makedepends="binutils-devel popt-devel avahi-libs-devel"
 short_desc="Distributed compilation for faster C/C++ builds"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://distcc.github.io"
-distfiles="https://github.com/distcc/${pkgname}/releases/download/v${version}/${pkgname}-${version}.tar.gz"
+distfiles="https://github.com/distcc/distcc/releases/download/v${version}/distcc-${version}.tar.gz"
 checksum=bead25471d5a53ecfdf8f065a6fe48901c14d5008956c318c700e56bc87bf0bc
 conflicts="chroot-distcc>=0"
 
-subpackages="distcc-gtk"
-
 CFLAGS="-fcommon"
 
+subpackages=""
```

Comment:
This doesn't work, empty subpackages will be calculated from template. `subpackages=" "` works but ...